content/docs/intro/concepts/state.md

```diff
@@ -13,124 +13,81 @@ menu:
 aliases: ["/docs/reference/state/"]
 ---
 
-## State
+Pulumi stores metadata about your infrastructure so that it can manage your cloud resources. This metadata is called _state_ and is how Pulumi knows when and how to create, read, delete, or update cloud resources. Pulumi stores this metadata, the state, in a _backend_ of your choosing. Options include the Pulumi Service, a secure and reliable hosted application with policies and safeguards to facilitate team collaboration, in addition to object storage in AWS, Azure, GCP, or a simple filesystem. Most of the time, you don't need to explicitly worry about state and backends, but in some cases you may.
 
-Pulumi stores its own copy of the current state of your infrastructure. This is
-often simply called _state_, and is stored in transactional snapshots we call
-_checkpoints_. A _checkpoint_ is recorded by Pulumi at various points so that
-it can operate reliably---whether that means diffing goal state versus
-current state during an update, recovering from failure, or destroying
-resources accurately to clean up afterwards. Because _state_ is critical to how
-Pulumi operates, we'll cover a few of the state backend options on this page.
+> Pulumi state does not include your cloud credentials. Credentials are kept local to your client &mdash; wherever the CLI runs &mdash; even when using the managed Pulumi Service backend. Pulumi _does_ store configuration and secrets, but encrypts those secrets using your chosen encryption provider. To learn more, see [Configuration and Secrets]({{< relref "config" >}}).
 
 ## Backends
 
-Pulumi supports multiple  _backends_ for storing your infrastructure
-state:
+Pulumi supports two classes of  _backends_ for storing your infrastructure state:
 
-- The Pulumi Service backend
-- A self-managed backend, either stored locally on your filesystem or remotely using
-a cloud storage service
+- **Service**: a managed cloud experience using the online or self-hosted Pulumi Service application
+- **Self-Managed**: an object store on your local filesystem, or in AWS, Azure, or Google Cloud
 
-### Pulumi Service backend
+Pulumi's open source SDK works great with all backends, although some details differ between them.
```

Comment:
Why mention `open source` here?

Reply:
I am just conscious that most people scrutinizing this page will be more sensitive to the open source nature, and wanted to reinforce that the SDK is indeed open source. However, I can drop it, definitely semi-superfluous.

Reply:
Now I understand :) Thanks for that!